include/mbedtls/x509_csr.h

```diff
@@ -58,6 +58,10 @@ typedef struct mbedtls_x509_csr {
 
     mbedtls_pk_context pk;          /**< Container for the public key context. */
 
+    unsigned int key_usage;     /**< Optional key usage extension value: See the values in x509.h */
+    unsigned char ns_cert_type; /**< Optional Netscape certificate type extension value: See the values in x509.h */
+    mbedtls_x509_sequence subject_alt_names;    /**< Optional list of raw entries of Subject Alternative Names extension (currently only dNSName and OtherName are listed). */
```

Comment:
It's not that other subtypes are not listed. It's just that these are parsed and respective structures (from the union) get filled, but for other types it's just pointers to raw data.

Reply:
This still needs to be updated.